packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+_Individuele formulier elementen en hun specifieke richtlijnen zijn ondergebracht in de componentenbibliotheek._
 
-## Introduction
+Met een formulier nodig je iemand uit tot een dialoog met jouw organisatie. Het is niet slechts een manier om jouw database te vullen maar een onderdeel van de gehele reis die iemand door jouw product maakt. Ontwerp een formulier vanuit degene die deze in moet vullen.
 
-A form is the moment the user enters into a dialogue with our products and organisation. It should therefore be as clear and convenient as possible.
+Iemand komt voor wat er zich aan de andere kant van het formulier bevindt; een antwoord op een vraag, een besteld product, een ingelogde staat, etc. Een formulier is een barrière tussen hen en dit doel. Het moet zo duidelijk mogelijk zijn hoe iemand dit doel kan bereiken en zo makkelijk mogelijk om te voltooien.
 
-A good form invites users to complete a task successfully.
+## Vormgeving en structuur
 
-## General guidelines
+### Opbouw
 
-Only ask for the strictly necessary information and inform users of the necessity of the requested information. Avoid the need for users to enter the same information several times.
+Voorkom zoveel mogelijk visuele afleiding op een formulierenpagina. Leidt iemand niet af van het primaire doel.
 
-### Logical structure
+#### Informatie dichtheid
 
-Structuring a form vertically is more pleasant and comfortable for users to both read and interact with.
+Bij het structureren van een formulier dient er voldoende ruimte te zijn tussen verschillende visuele elementen. Deze elementen dienen zo gegroepeerd te zijn dat het duidelijk is dat bijvoorbeeld een label bij een tekstinvoerveld hoort.
 
-Users scan and read a page on a screen according to an ‘F’ pattern (in LTR languages). They first start scanning horizontally at the top and then visually down a page vertically, occasionally making a horizontal excursion.
+Houdt rekening met verschillende staten van elementen; een foutmelding zou bijvoorbeeld in de buurt van de foutieve invoer moeten staan en een directe relatie moeten tonen.
 
-Forms in a one-column layout are easier to scan. A logical sequence indicates which data is expected at what time.
+### Groeperen
 
-Indicating the length of the expected input can (sparingly) be shown visually by input fields of different widths. For instance, data for a house number is often shorter than a telephone number. An exception to this rule can be made and a house number addition field can be displayed next to a house number field. This is however only possible if the fields have a direct relationship with each other and are in a logical order.
+Wanneer een formulier als dialoog wordt gezien kunnen verschillende onderwerpen natuurlijk ontstaan. Gebruik deze onderwerpen om een formulier logisch te groeperen.
+NAW-gegevens zijn bijvoorbeeld een logisch onderwerp die te groeperen is. Ook ondertekening bestaande uit bijvoorbeeld datum, naam en plaats zijn zo’n logische groep.
```

Comment:
Meervoud en enkelvoud loopt een beetje door elkaar heen hier voor mijn gevoel, al kan ik m'n vinger er niet helemaal op leggen. Misschien meer in de richting van "NAW-gegevens kun je bijvoorbeeld zien als een groep met een gedeeld onderwerp. Net als ondertekening waar datum, naam en plaats samen zo'n logische groep vormen."

Reply:
Ik denk dat het simpel te fixen is: "NAW-gegevens [...] is → zijn" en "Ook ondertekening [...] zijn → is zo'n groep"